Tests/FluentAssertions.Specs/Primitives/HttpResponseMessageAssertionSpecs.cs

```diff
@@ -0,0 +1,261 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Net;
+using System.Net.Http;
+using Xunit;
+using Xunit.Sdk;
+
+namespace FluentAssertions.Specs.Primitives
+{
+    public class HttpResponseMessageAssertionSpecs
+    {
+        [Theory]
+        [MemberData(nameof(GetSuccessStatusCodes))]
+        public void Should_succeed_when_asserting_statuscode_is_successful(HttpStatusCode statusCodeOfResponse)
```

Comment:
🙃 I'm nitpicking here, but `statuscode` consists of two separate words. I would also remove the `asserting` word to make the functional names more concise